tox.ini

```diff
@@ -0,0 +1,21 @@
+[tox]
+skipsdist = true
+envlist = tests, flake8
+
+[testenv]
+passenv = PYTEST_ADDOPTS="--variables=/path/to/variables.json" tox
```

Comment:
`passenv` is just a list of environment variable names to pass through, so this should just be `PYTEST_ADDOPTS`.